outputs.tf

```diff
@@ -14,7 +14,42 @@
  * limitations under the License.
  */
 
-output "buckets_list" {
-  value       = "${compact(split(" ", data.external.bucket_retrieval.result["buckets_list"]))}"
-  description = "The buckets list within the created project"
+output "forseti-server-vm-name" {
+  description = "Forseti Server role vm name"
+  value       = "${google_compute_instance.forseti-server.name}"
+}
+
+output "forseti-server-vm-ip" {
+  description = "Forseti Server role ip address"
+  value       = "${google_compute_instance.forseti-server.network_interface.address}"
```

Comment:
I'm receiving the following errors:
```
* module.forseti.output.forseti-server-vm-ip: Resource 'google_compute_instance.forseti-server' does not have attribute 'network_interface.address' for variable 'google_compute_instance.forseti-server.network_interface.address'
* module.forseti.output.forseti-client-vm-ip: Resource 'google_compute_instance.forseti-client' does not have attribute 'network_interface.address' for variable 'google_compute_instance.forseti-client.network_interface.address'
```

The Terraform docs say to use `network_interface.network_ip` instead:

> address - (Optional, Deprecated) The private IP address to assign to the instance. If empty, the address will be automatically assigned. This attribute has been deprecated. Use network_interface.network_ip instead.

https://www.terraform.io/docs/providers/google/r/compute_instance.html

Reply:
+1 

Found that the value works when it is: "${google_compute_instance.forseti-server.network_interface.0.network_ip}"